### PR TITLE
Style editor toolbar

### PR DESCRIPTION
**Pull request by alebedev**
# Notion Ticket: [CLX-XXX]

## What?
Apply styles to rich text editor toolbar

## Why?
- Make it look better
- Test styling approach with css modules + className

## If applicable, rooms for improvement?
- 

## Optional checklist
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1214?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #1208** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1208?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #1209** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1209?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #1210** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1210?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #1212** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1212?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #1214** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1214?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
            * **PR #1215** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1215?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1214?utm_source=stack-comment).

File: src/portals/hope/features/claims/new/richTextEditor.module.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.richTextEditor {
+    --baseBg: #fafafa;
+}
```
Comment (guilhermespopolin) [inline src/portals/hope/features/claims/new/richTextEditor.module.css:3]: Is this CSS variable being used by MDXEditor? 🤔

**Review by guilhermespopolin — APPROVED** [review]

File: src/portals/hope/features/claims/new/richTextEditor.module.css
Comment (falklinus) [inline src/portals/hope/features/claims/new/richTextEditor.module.css:1]: I'm thinking we could move this into the `EditorWrapper` css, or maybe that didn't work? 🤔

**Review by falklinus — APPROVED** [review]

File: src/portals/hope/features/claims/new/richTextEditor.module.css
Comment (alebedev) [reply src/portals/hope/features/claims/new/richTextEditor.module.css:1]: CSS variables would be shadowed by ones inserted on editor root element

File: src/portals/hope/features/claims/new/richTextEditor.module.css (hunk 1 above)
Comment (alebedev) [reply src/portals/hope/features/claims/new/richTextEditor.module.css:3]: Yes

**Comment by alebedev** [comment]
### Merge activity

* **Jan 29, 10:07 AM**: [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/hedgehog/1214) rebased this pull request after merging its parent, because this pull request is set to merge when ready.

**Pull request merged**